yotta/data/example-authcrypt/README.md

```diff
@@ -1,68 +0,0 @@
-# Authenticated Encryption Example
```

Comment:
Consider moving the relevant information here into [the README.md in the new examples repo](https://github.com/ARMmbed/mbed-os-example-tls/blob/master/authcrypt/README.md).

Reply:
I personally think that this information is already in the new example repo, but most of it is in the [main README.md](https://github.com/ARMmbed/mbed-os-example-tls) instead of the example because otherwise the information would just be duplicated.

Reply:
I think the examples are already sufficiently well documented, as @andresag01 states, this information is in the main example `README.md`.